src/framework/mlt_slices.h

```diff
@@ -38,4 +39,12 @@ extern int mlt_slices_count( mlt_slices ctx );
 
 extern mlt_slices mlt_slices_init_pool( int threads, int policy, int priority, const char* name );
 
+extern mlt_slices mlt_slices_get_global( mlt_schedule_policy policy );
```

Comment:
I would question the need for this function to be public. Is it just so that it is possible to call mlt_slices_count()? If so, just make a new version of mlt_slices_count() that returns the global number of slices. It could be called:

    extern int mlt_slices_count_global();

It think it would be valid for the implementation to never provide the global contexts to the public.